contract/contracts/authRole.sol

```diff
@@ -0,0 +1,128 @@
+// SPDX-License-Identifier: MIT
+pragma solidity 0.5.x;
+
+import "./auditLog.sol";
```

Comment:
prefix `./` is not necessary.

Reply:
If I remove the `./` then I will meet the following error, `ParserError: Source "auditLog.sol" not found: File import callback not supported
import "auditLog.sol";`